pkg/injector/patcher/sidecar_volumes.go

```diff
@@ -78,7 +77,7 @@ func (c *SidecarConfig) getTokenVolume() corev1.Volume {
 				DefaultMode: ptr.Of(int32(420)),
 				Sources: []corev1.VolumeProjection{{
 					ServiceAccountToken: &corev1.ServiceAccountTokenProjection{
-						Audience:          securityConsts.ServiceAccountTokenAudience,
+						Audience:          c.SentrySPIFFEID.String(),
```

Comment:
Can you make this an option passed to SidecarConfig as a string? For example just "Audience" and then the caller will pass a SPIFFE ID. Since the injector can be used as a library

Reply:
We need to use the spiffe library as it is important that the Sentry SPIFFE ID is validated as a well formed string. It makes little difference to the go modules of the injector when used as a library. The SPIFFE library itself is small.

```
$ diff go.mod go.mod_
12a13
> 	github.com/spiffe/go-spiffe/v2 v2.1.6
$ diff go.sum go.sum_
131a132,133
> github.com/spiffe/go-spiffe/v2 v2.1.6 h1:4SdizuQieFyL9eNU+SPiCArH4kynzaKOOj0VvM8R7Xo=
> github.com/spiffe/go-spiffe/v2 v2.1.6/go.mod h1:eVDqm9xFvyqao6C+eQensb9ZPkyNEeaUbqbBpOhBnNk=
```

Reply:
Yes I am not saying don't pass the SPIFFE ID, I am just saying make `c.SentrySPIFFEID` of type `String()`. Then do the `.String()` call before passing the value to SidecarConfig.

Reply:
Gotcha, done